WordPress/Classes/ViewRelated/Site Creation/Wizard/SiteCreationWizardLauncher.swift

```diff
@@ -57,5 +56,24 @@ final class SiteCreationWizardLauncher {
 
     init(onDismiss: ((Blog, Bool) -> Void)? = nil) {
         self.onDismiss = onDismiss
+
+        let siteIntentVariant = SiteIntentAB.shared.variant
+        let siteNameVariant = ABTest.siteNameV1.variation
+
+        SiteCreationWizardLauncher.fireInitEvents(siteIntentVariant: siteIntentVariant, siteNameVariant: siteNameVariant)
```

Comment:
Events to track which A/B group were living in the `init` functions of the steps, but those may not be called now if the orderer determines the view isn't needed. I'm open to going in other directions with this.